application/src/test/kotlin/yeoksam/fc/application/user/service/UserServiceTest.kt

```diff
@@ -0,0 +1,46 @@
+package yeoksam.fc.application.user.service
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import yeoksam.fc.application.TestApplication
+import yeoksam.fc.application.user.dto.SaveUserCommand
+import yeoksam.fc.domain.user.model.UserRole
+import yeoksam.fc.domain.user.vo.Email
+import yeoksam.fc.infrastructure.user.repository.UserEntityRepository
+import yeoksam.fc.template.IntegrationTestTemplate
+
+@SpringBootTest(classes = [TestApplication::class])
```

Comment:
```suggestion
@SpringBootTest
```

흠,, 혹시 이거 떼면 문제 생기나요 🤔 ????
일단 제가 떼고 실행했을땐 잘 되긴 하는데,,,

Reply:
아아 이건 빼도 됩니다!! 테스트 적용해보려고, SpringBootApplication 위치를 바꿔가며 테스트하다가 딱히 지우진 않았네요!!ㅎㅎ
지워놓겠습니다! 👍👍